### PR TITLE
Split RHEL 7 CIS profile - section 5

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- update the policy file
- add or correct cis references in rule.yml files

#### Rationale:

CIS split effort

### CODE REVIEW

**Review by yuumasato — CHANGES_REQUESTED** [review]
Made some general comments.
The only change I see as essential is the `pam_unix.so` with `remember`.

File: controls/cis_rhel7.yml [hunk 1]
```diff
@@ -1543,3 +1543,483 @@ controls:
...
+    - set_password_hashing_algorithm_systemauth
+
+  - id: 5.4.4
+    title: Ensure password reuse is limited (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: yes
+    notes: |-
+      Usage of pam_unix.so module is not deprecated and is not supported by this policy interpretation.
```
Comment (yuumasato) [inline controls/cis_rhel7.yml:1920]: I think you meant that use of pam_unix.so with remember is deprecated.

**Review by yuumasato — COMMENTED** [review]
LGTM
@vojtapolasek please rebase and fix the conflicts, :)

**Comment by vojtapolasek** [comment]
Thank you for the review. I think we do not have rule for checking of SSH exchange key algorithms as you suggested in one of your comments.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**